_includes/head.html

```diff
@@ -4,10 +4,17 @@
     <title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
 
     <link rel="icon" href="{{ site.baseurl }}/assets/images/favicon.ico">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="description" content="{{ site.description }}">
     <link rel="canonical" href="{{ page.url | replace:'index.html','' | prepend: site.baseurl | prepend: site.url }}">
 
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
```

Comment:
I suspect this is here because of the script tag [in line 154 of en/facilitate/index.html](https://github.com/p2pu/p2pu-website/pull/52/files#diff-14b49e7c9e1e8d4ecfa9cb7b1800828eR154).

Rather put the javascript code in a separate file and include using `extra_js` in the Jekyll frontmatter. See [here](https://github.com/p2pu/p2pu-website/blob/gh-pages/_includes/scripts.html#L18-L26) on how `extra_js` is handled.

Putting JQuery in the head will block page loading until the JavaScript has been downloaded which sucks on a mobile.